src/libraries/Common/tests/System/IO/Compression/CompressionStreamUnitTestBase.cs

```diff
@@ -441,6 +441,31 @@ public async Task BaseStream_ValidAfterDisposeWithTrueLeaveOpen(CompressionMode
             else
                 baseStream.Read(bytes, 0, size);
         }
+
+        [Theory]
+        [MemberData(nameof(UncompressedTestFiles))]
+        public async Task CompressionLevel_SizeInOrder(string testFile)
+        {
+            using var uncompressedStream = await LocalMemoryStream.readAppFileAsync(testFile);
+
+            async Task<long> GetLengthAsync(CompressionLevel compressionLevel)
+            {
+                using var mms = new MemoryStream();
+                using var compressor = CreateStream(mms, compressionLevel);
+                await uncompressedStream.CopyToAsync(compressor);
+                compressor.Flush();
+                return mms.Length;
+            }
+
+            long noCompressionLength = await GetLengthAsync(CompressionLevel.NoCompression);
```

Comment:
From this comment:https://github.com/dotnet/runtime/pull/41960#issuecomment-690781119

We  cannot guarantee size order. This test should instead just verify that the contents of the generated compressed files are not corrupt.

Reply:
@danmosemsft any agreement please?

Reply:
It does seem likely though that size levels increase monotonically -- that would otherwise break my expectations -- my suggestion is to keep the test as is, and if it breaks at some future point, we can just loosen the test then. does that seem reasonable @carlossanlop ?

Reply:
Fine by me. @ericstj ?

Reply:
Sounds reasonable.